src/index.html

```diff
@@ -4,9 +4,289 @@
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
-    <link rel="stylesheet" href="./styles/main.scss">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="./styles/blocks/main.scss">
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <a href="#home" class="header__logo">
+        <img
+          class="header__logo-img"
+          src="./images/logo.png"
+          alt="Logo">
+      </a>
+      <nav class="header__nav">
+        <ul class="header__nav-list">
+          <li class="header__nav-item">
+            <a href="#" class="header__nav-link is-active">Apple</a>
+          </li>
+          <li class="header__nav-item">
+            <a href="#" class="header__nav-link">Samsung</a>
+          </li>
+          <li class="header__nav-item">
+            <a href="#" class="header__nav-link">Smartphones</a>
+          </li>
+          <li class="header__nav-item">
+            <a
+              href="#"
+              class="header__nav-link"
+              data-qa="nav-hover"
+            >
+            Laptops & Computers
```

Comment:
Please fix it 
![image](https://github.com/mate-academy/layout_catalog/assets/70009205/b0230583-f056-4c1c-a555-c215c6b638c0)